spec/features/result_spec.rb

```diff
@@ -0,0 +1,23 @@
+# a winner will be declared
+require_relative 'web_helpers'
+
+feature 'want to see the result' do
+
+  scenario 'it is a draw' do
+      sign_in
+      draw
+      expect(page).to have_content "it's a draw"
+  end
+
+  scenario 'player 1 wins' do
+      sign_in
+      winner_1
```

Comment:
for `winner_1` and `winner_2` I'd probably not use helper methods because that code is critical to one's understanding of what's being tested... but other people might disagree!